src/plugins/files_management/public/i18n_texts.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const i18nTexts = {
+  titleColumnName: i18n.translate('filesManagement.table.titleColumnName', {
+    defaultMessage: 'Name',
+  }),
+  tableListTitle: i18n.translate('filesManagement.table.title', { defaultMessage: 'Files' }),
+  entityName: i18n.translate('filesManagement.entityName.title', { defaultMessage: 'file' }),
+  retry: i18n.translate('filesManagement.button.retry', {
+    defaultMessage: 'Retry',
+  }),
+  entityNamePlural: i18n.translate('filesManagement.entityNamePlural.title', {
+    defaultMessage: 'files',
+  }),
+  emptyPromptTitle: i18n.translate('filesManagement.emptyPrompt.title', {
+    defaultMessage: 'No files found',
+  }),
+  emptyPromptDescription: i18n.translate('filesManagement.emptyPrompt.description', {
+    defaultMessage: 'Any files created in Kibana will be listed here.',
```

Comment:
Could the description instead provide advice on how to create a file?

Reply:
It is a little bit tricky because we don't want to drive file creation from this UI. The idea will be for files to be created in any number of the solutions like Analytics (dashboard, canvas), Security (cases) or Observability... Do you have any idea how we could communicate that?